go.mod

```diff
@@ -94,7 +94,7 @@ require (
 	k8s.io/kubernetes v1.22.10
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
 	oras.land/oras-go v1.2.0
-	oras.land/oras-go/v2 v2.0.0-20220621073716-14422086e418
+	oras.land/oras-go/v2 v2.0.0-alpha
```

Comment:
They released `v2.0.0-rc.1`, perhaps you want to use it instead.

Reply:
yes, I am on it. Next PR